packages/batcher/webserver/src/index.ts

```diff
@@ -451,6 +460,45 @@ async function initializeServer(
   );
 }
 
+//
+// This method executes Google reCaptcha3 validation for backend.
+// If the validation fails, an error is thrown - otherwise, the method returns void.
+// To enable validation:
+//  1. Create a reCaptcha3 account and get the site key and secret key. (https://www.google.com/recaptcha)
+//  2.a Set the RECAPTCHA_V3_BACKEND in the .env.<NETWORK> file with your secret key.
+//  2.b Set the RECAPTCHA_V3_FRONTEND in the .env.<NETWORK> file with your site key.
+//  3. Add the reCaptcha3 script to the frontend or call injectReCaptchaToHTML()
+async function reCaptchaValidation(captcha: string): Promise<void> {
+  if (!ENV.RECAPTCHA_V3_BACKEND) return;
+  const recapchaURL = 'https://www.google.com/recaptcha/api/siteverify';
+  /**
+   * API Request Parameters:
+   *   secret	  Required. The shared key between your site and reCAPTCHA.
+   *   response	Required. The user response token provided by the reCAPTCHA client-side integration on your site.
+   *   remoteip	Optional. The user's IP address.
+   */
+  const recaptchaResponse = await axios({
+    url: recapchaURL,
+    method: 'post',
+    headers: { 'content-type': 'application/x-www-form-urlencoded' },
+    data: {
+      secret: ENV.RECAPTCHA_V3_BACKEND,
+      response: captcha,
+    },
+  });
+  /* reCaptcha API Response example
```

Comment:
Do you want to define a proper Typescript type for this (maybe there is already type definitions you can download from somewhere? Otherwise, you can just define the type yourself even if it's not perfect)

Reply:
I didn't find a public type definition for this